Makefile

```diff
@@ -210,6 +211,8 @@ config.h:
 	echo '/* Default config.h generated by Makefile */' > $@
 	echo '#define HAVE_LIBBZ2 1' >> $@
 	echo '#define HAVE_LIBLZMA 1' >> $@
+	echo '#define HAVE_FSEEKO 1' >> $@
+	echo '#define HAVE_DRAND48 1' >> $@
```

Comment:
fseeko *is* used and drand48 is used by more than ks_shuffle.  Possibly we could rewrite the code that uses both, but as these are POSIX functions the intent is simply to put OS-specific deficiencies in one single place so we can maintain our position of desiring a POSIX compatible compilation.  It also makes the PR much simpler, and safer, as it doesn't contain a lot of rewriting of code just to make it run on Windows.